### PR TITLE
Suma payg part1

### PR DESCRIPTION
**Pull request by mcalmer**
## What does this PR change?

First part of SUMA PAYG feature.

Port of SUSE/spacewalk#21819
Obsolete https://github.com/uyuni-project/uyuni/pull/7173

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests" 
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Comment by jordimassaguerpla** [comment]
Hi! Sorry there was an issue with podman. The issue is fixed in master. Could you please rebase your PR? Sorry for any inconvenience.

File: web/html/src/manager/admin/payg-shared/info/payg-info-edit.tsx [hunk 1]
```diff
@@ -46,6 +46,7 @@ const PaygInfoEdit = (props: Props) => {
           setItem(props.payg);
           setErrors(null);
         }}
+        disableEditing={!props.editing}
```
Comment (Etheryte) [inline web/html/src/manager/admin/payg-shared/info/payg-info-edit.tsx:49]: Here and elsewhere where this name is used, I would recommend making the prop name consistent across the board. `payg.renderer.tsx` uses the name `readOnly` which seems to have the same purpose, I think that name is better.

**Review by Etheryte — APPROVED** [review]

File: web/html/src/manager/admin/payg-shared/info/payg-info-edit.tsx (hunk 1 above)
Comment (mackdk) [reply web/html/src/manager/admin/payg-shared/info/payg-info-edit.tsx]: Fixed. Could please double check that I haven't missed any places where the property needs to be updated?

**Comment by mackdk** [comment]
Something went wrong with the rebase

**Comment by juliogonzalez** [comment]
I am removing @uyuni-project/release-engineering from this PR as reviewes. Please add us again only when the PR is ready to review, as we are getting a lot of notifications and it seems you still need to rebase.

**Comment by jordimassaguerpla** [comment]
@mcalmer Hi! You need to rebase to be able to run the acceptance tests. They are failing now because of wrong podman version. Sorry for any inconvenience.

**Pull request merged**